rtt_ros2_params/README.md

```diff
@@ -0,0 +1,49 @@
+
+# Description
+
+ROS2 provides a slightly different infrastructure to deal with parameters. An introduction guide can be found in [Understanding ROS2 prameters](https://index.ros.org/doc/ros2/Tutorials/Parameters/Understanding-ROS2-Parameters/).
+The main difference is that there is no longer a centralized parameter server, but instead, parameters are maintained per node.
+
+## `ros2-params` service
```

Comment:
For backwards compatibility with [rtt_ros_integration](https://github.com/orocos/rtt_ros_integration) for ROS 1, using service names `rosparam`, `rosservice` and `rosnode` (not in ROS 1) would be another option.

Reply:
It sounds nice to use the same names `rosparam`, `rosservice`, `rosnode`. But I wonder, in a machine where both ROS 1 and ROS 2 are installed, it could generate some ambiguity. You can only choose which is used based on which package you are importing.

Reply:
Opened issue #18.